go/chat/storage/inbox.go

```diff
@@ -722,6 +770,12 @@ func (i *Inbox) Clear(ctx context.Context, uid gregor1.UID) (err Error) {
 	return nil
 }
 
+func (i *Inbox) ClearInMemory(ctx context.Context, uid gregor1.UID) (err Error) {
```

Comment:
unused?

Reply:
yes, but I thought it might be useful some day.